### DISCUSSION TITLE
How can I use BYG or BWG on 1.20.1?

### DISCUSSION THREAD

**Discussion by TheCrossCarrier** [CorgiTaco/BYG#1318] [Q&A] [answered]
How can I use BYG or BWG on 1.20.1?

Thanks <3

**Comment by CorgiTaco** [comment]
You'll need to join our patreon to access the current alpha files: https://www.patreon.com/biomesyougo


OR you'll need to wait for public release

**Reply by TheCrossCarrier** [reply]
What is the approximate date for the planned public release?

**Reply by lolpowerluke** [reply]
well since the alpha just started (there are only 2 versions), I'd say it'll at least take a few months before the beta even starts
but idk, I'm not a dev for this project

**Reply by CorgiTaco** [reply, marked as answer]
We're aiming for before the end of the month
